ta4j-core/src/main/java/org/ta4j/core/indicators/StochasticRSIIndicator.java

```diff
@@ -52,23 +53,23 @@ public StochasticRSIIndicator(TimeSeries series, int barCount) {
 
     /**
      * Constructor.
-     * @param indicator the indicator
+     * @param closePriceIndicator the close prices
      * @param barCount the time frame
      */
-    public StochasticRSIIndicator(Indicator<Num> indicator, int barCount) {
-        this(new RSIIndicator(indicator, barCount), barCount);
+    public StochasticRSIIndicator(Indicator<Num> closePriceIndicator, int barCount) {
```

Comment:
Why renaming to CosePriceIndicator? StochRSI could also be based on High, Low, Open or so

Reply:
> Why renaming to CosePriceIndicator? StochRSI could also be based on High, Low, Open or so

But in practice it never is, and the ambiguity has generated confusion that has spawned multiple Issues.  And this is a semantic change so you can still use any `Indicator`.  Users need guidance when they see `Indicator<Num> indicator` and `@param indicator the Indicator` as a parameter or they will use it wrong and open Issues.  However, I have moved the hint to the javadoc comment to get the PR accepted.